pyqtgraph/graphicsItems/PlotDataItem.py

```diff
@@ -1214,7 +1214,7 @@ def dataType(obj):
             elif obj.ndim == 2 and obj.dtype.names is None and obj.shape[1] == 2:
                 return 'Nx2array'
             else:
-                raise Exception('array shape must be (N,) or (N,2); got %s instead' % str(obj.shape))
+                raise TypeError('array shape must be (N,) or (N,2); got %s instead' % str(obj.shape))
```

Comment:
Hi, this one should be a `ValueError` not a `TypeError` 👍🏻

Reply:
OMG sorry about that :rofl: